src/plugins/harmonies.ts

```diff
@@ -0,0 +1,42 @@
+import { Plugin } from "../extend";
+
+export type Harmony =
+  | "analogous"
+  | "complimentary"
+  | "rectangle"
+  | "tetradic"
+  | "triadic"
+  | "splitcomplimentary";
+
+declare module "../colord" {
+  interface Colord {
+    /**
+     * Returns an array of harmony colors as `blossom` instances.
```

Comment:
:)
```suggestion
     * Returns an array of harmony colors as `Colord` instances.
```

P.S. I can mention your project in changelog if you want :)

Reply:
Haha, yes, I tried to implement it in my "clone" :) No need, really, it is not that good :) I will fix it.